scripts/version/set-version.ps1

```diff
@@ -15,6 +15,13 @@ Param(
 Set-StrictMode -version 2.0
 $ErrorActionPreference = "Stop"
 
+function Set-VersionForJava() {
+    Write-Header "Updating version in Java files"
+
+    mvn org.codehaus.mojo:versions-maven-plugin:2.2:set "-DnewVersion=${fixedVersion}-SNAPSHOT" -DgenerateBackupPoms=false -B -e
```

Comment:
On sonar-dotnet side, this updates version of the main project and its submodules.

But ITs are not versioned at all, because they are not a referenced submodule. For consistency and simplicity, I'd prefer to have version `1.0-SNAPSHOT` here, the same as on `sonar-dotnet` side. 
https://github.com/SonarSource/sonar-dotnet/blob/f0e16b3bf40965aee8c861f96a84c586def0eafd/its/pom.xml#L15

Or remove it, if we can (not sure)